doc/source/about.rst

```diff
@@ -18,9 +18,10 @@ data types can be defined. This allows *NumPy* to seamlessly and
 speedily integrate with a wide variety of databases.
 
 NumPy is a successor for two earlier scientific Python libraries:
-NumPy derives from the old *Numeric* code base and can be used
-as a replacement for *Numeric*.  It also adds the features introduced
-by *Numarray* and can also be used to replace *Numarray*.
+NumPy derives from the :doc:`old Numerice <reference/routines.oldnumeric>` code
+base and can be used as a replacement for *Numeric*.  It also adds the features
```

Comment:
It is not a drop in replacement at this point, as the compatibility module has been removed.

Reply:
We should probably put the compatibility module somewhere.

Reply:
reworded